mmcv/runner/checkpoint.py

```diff
@@ -503,7 +504,8 @@ def load_checkpoint(model,
                     filename,
                     map_location=None,
                     strict=False,
-                    logger=None):
+                    logger=None,
+                    revise_keys=[(r'^module.', '')]):
```

Comment:
@MeowZheng Do you prefer a dict `{r'^module.': ''}` over the current list of tuple?

Reply:
Some previous version python doesn't preserve the order for `dict`, and it may be not safe to revise state_dict, but key-value stores are more understandable. In this way, I prefer `OrderedDict`. :)

Reply:
`OrderedDict` is too long...

Let's keep the list